modules/deck.gl-geoarrow/src/index.ts

```diff
@@ -1 +1 @@
-export { GeoArrowPointLayer } from "./point.ts.bak";
+export { GeoArrowPointLayer } from "./point.js";
```

Comment:
The `.js` extensions are essential for relative imports so that TypeScript compiler will output true ESM. TypeScript won't touch imports, so you need to provide the extension in the output.